### PR TITLE
set shouldPresentInFullscreen as true

### PR DESCRIPTION
**Pull request by lcanalimp**
set shouldPresentInFullscreen as true due iOS 17 known problem

### CODE REVIEW

**Review by 39otrebla — APPROVED** [review]

**Review by 39otrebla — CHANGES_REQUESTED** [review]
Wrong target repo/branch.

File: Sources/BottomSheet/Core/Presentation/BottomSheetPresentationController.swift [hunk 1]
```diff
@@ -150,7 +150,7 @@ public final class BottomSheetPresentationController: UIPresentationController {
     }
 
     public override var shouldPresentInFullscreen: Bool {
-        false
+        true
```
Comment (mikhailmaslo) [inline Sources/BottomSheet/Core/Presentation/BottomSheetPresentationController.swift:153]: Should we set it only for IOS 17?

I'm not sure how this flag affects older versions
```Suggestion
        if #available(iOS 17.0, *) {
            return true
        } else {
            return false
        }
```

**Comment by mikhailmaslo** [comment]
Might address this issues:
- https://github.com/joomcode/BottomSheet/issues/32
- https://github.com/joomcode/BottomSheet/issues/31

**Comment by mikhailmaslo** [comment]
Finished the same code here https://github.com/joomcode/BottomSheet/pull/38

**Pull request closed without merging**